graphbolt/include/graphbolt/threading_utils.h

```diff
@@ -0,0 +1,30 @@
+/**
+ *  Copyright (c) 2023 by Contributors
+ * @file graphbolt/utils.h
+ * @brief Threading utils in graphbolt.
+ */
+#ifndef GRAPHBOLT_THREADING_UTILS_H_
+#define GRAPHBOLT_THREADING_UTILS_H_
+
+#include <omp.h>
+
+namespace graphbolt {
+namespace runtime {
+
+inline size_t compute_num_threads(size_t begin, size_t end, size_t grain_size) {
+#ifdef _OPENMP
```

Comment:
will we assume `GraphBolt` utilize OPENMP for parallel in most of time? Does `torch::parallel_for` utilize OpenMP? Will we prefer to use `torch::parallel_for`? If so, do we need to add `#ifdef` here?